src/modm/board/rp_pico/board.xml

```diff
@@ -0,0 +1,17 @@
+<library>
+  <repositories>
+    <repository>
+      <path>../../../../repo.lb</path>
+    </repository>
+  </repositories>
+
+  <options>
+    <option name="modm:target">rp2040</option>
+    <option name="modm:platform:core:boot2">w25q080</option>
+    <option name="modm:platform:multicore:core_storage.0">modm_section(".scratch_y_data")</option>
+    <option name="modm:platform:multicore:core_storage.1">modm_section(".scratch_x_data")</option>
```

Comment:
I feel like we could just default place it into `.cpu0_data`, since it's only a few bytes of memory anyways (I think it's 8B per scheduler?)